pages/index.rst

```diff
@@ -51,8 +51,8 @@ releases so far, `0.0.4`__ being the most recent one.
 
 .. __: https://hpyproject.org/blog/posts/2022/06/hpy-0.0.4-third-public-release/
 
-The Python/C API is huge and at the moment only a small fraction of it is
-available in HPy. We are experimenting by porting existing extensions to HPy
+The Python/C API is huge and at the moment the most popular functions are
+available. We are experimenting by porting existing extensions to HPy
```

Comment:
I seem to recall @antocuni did an analysis of the code in https://github.com/hpyproject/top4000-pypi-packages, can we create a progress report of api-function-name, #times it is used in the packages, checkbox if implemented in HPy

Reply:
We used to do this for the [micronumpy port](https://buildbot.pypy.org/numpy-status/26e09b343fc3329dab0fbd12e8afb31af4a35808.html) of NumPy in PyPy

Reply:
If we made a list we could also mark ones that we aren't necessarily intending to port.

Reply:
I made this script some time ago based on one of @antocuni's scripts to analyze top packages with C API usage. It's more focused to find "problematic" usage and give an overview of how many lines of code there would be to touch for an HPy port, but it may be useful to generate a list with: https://gist.github.com/timfel/3fc3b1f339917ff48b69289e79a39f8d

I can take a look tomorrow to adapt it to create such a list.

Reply:
Cool. Maybe it could be a page we link to here in the future